src/inject/dynamic-theme/image.ts

```diff
@@ -49,7 +50,12 @@ async function urlToImage(url: string) {
     });
 }
 
+const ImageCache = new Map();
```

Comment:
These results are already cached, see [here](https://github.com/darkreader/darkreader/blob/master/src/inject/dynamic-theme/modify-css.ts#L227).

So, I think this PR has no effect.

Anyway some comments:
- Start variables from lowercase.
- Remember to clean the cache when necessary.